extended-utxo-spec/extended-utxo-specification.tex

```diff
@@ -627,8 +633,6 @@ \subsection{Validity of EUTXO-1 transactions}
   \multicolumn{3}{l}{\lookupTx{\_}{\_} : \s{Ledger} \times \TxId \rightarrow \eutxotx{}}\\
```

Comment:
You can remove the underscores now and define it like a normal function.

Reply:
Well spotted.